packages/tychon/data_stream/tychon_cve/elasticsearch/ingest_pipeline/default.yml

```
@@ -13,6 +13,18 @@ processors:
   - set:
       field: _id
       value: "{{id}}"
+  - set:
+      field: tychon.ipv4
+      value: "{{host.ipv4}}"

```

```suggestion
        - set:
            field: tychon.ipv4
            copy_from: host.ipv4
            ignore_empty_value: true

```

This way you don't have empty/null values into `tychon.ipv4`